examples/ipfs-kad/src/main.rs

```diff
@@ -18,10 +18,7 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-//! Demonstrates how to perform Kademlia queries on the IPFS network.
-//!
-//! You can pass as parameter a base58 peer ID to search for. If you don't pass any parameter, a
-//! peer ID will be generated randomly.
+#![doc = include_str!("../README.md")]
```

Comment:
I think last time I tried this one, it was missing the `rsa` feature flag and thus we could not connect to the IPFS bootstrap nodes. Has this been fixed?

Reply:
@thomaseizinger Don't think so because when I run the example with `cargo run QmNnooDu7bfjPFoTZYxMNLWUQJyrVwtbZg5gBMjTezGAJN` where `QmNnooDu7bfjPFoTZYxMNLWUQJyrVwtbZg5gBMjTezGAJN` is one of the bootstrap node, I get the answer:
```sh
Searching for the closest peers to QmbLHAnMoJPWSCR5Zhtx6BHJX9KiKNN6tpvbUcqanj75Nb
Query finished with no closest peers.
```

Reply:
Right, if you add the `rsa` feature flag in `Cargo.toml`, you should get much more interesting results :)

Reply:
@thomaseizinger Done